Marlin/src/pins/pins.h

```diff
@@ -507,7 +507,9 @@
 #elif MB(MKS_ROBIN_MINI)
   #include "stm32f1/pins_MKS_ROBIN_MINI.h"      // STM32F1                                env:mks_robin_mini
 #elif MB(MKS_ROBIN_NANO)
-  #include "stm32f1/pins_MKS_ROBIN_NANO.h"      // STM32F1                                env:mks_robin_nano
+  #include "stm32f1/pins_MKS_ROBIN_NANO.h"      // STM32F1                                env:mks_robin_nano env:mks_robin_nano35
+#elif MB(MKS_ROBIN_NANO_V2)
+  #include "stm32f1/pins_MKS_ROBIN_NANO_V2.h"   // STM32F1                                env:mks_robin_nano env:mks_robin_nano35
```

Comment:
Should this refer at all to `env:mks_robin_nano35_v2` or is that env experimental?

Reply:
Its an open question yet. Currently, the only difference from the nano 1.2 and nano 2 env is the output firmware file name. We are about to do some tests, to check what works and what don't.